extensions/wikia/CommunityPage/styles/components/_variables.scss

```diff
@@ -8,6 +8,8 @@ $community-page-font-family: "HelveticaNeue", Helvetica, Arial, sans-serif;
 $module-default-line-height: 20px;
 $avatar-size: 30px;
 $avatar-flex: 0 0 $avatar-size;
+$username-column-width: 174px;
+$userrank-column-width: 26px;
```

Comment:
I thought it would be safe in case if we want to change the width of right rail.
`userrank-column-width` is used only once but I put it here because it relates to `username-column-width`